common/metadata-helper.cpp

```diff
@@ -229,7 +231,10 @@ namespace rs2
                         CloseHandle(sei.hProcess);
                         if (exitCode)
                             throw std::runtime_error("Failed to set metadata registry keys!");
-                        return true;
+                        // returning false here so that the instandce that runs "not as admin"
+                        // will not even try to do the writing to registry job
+                        // This job is done by the "run as admin" instance.
+                        return false;
```

Comment:
We have an `if` and an `else` both returning false?
and more code after? return true?
Take another look please..